src/promptflow/tests/executor/unittests/_utils/test_tool_utils.py

```diff
@@ -69,6 +70,13 @@ def func(input_str: str):
             function_to_interface(func, {"input_str": "test"})
         assert "Duplicate inputs found from" in exec_info.value.args[0]
 
+    def test_function_to_interface_with_kwargs(self):
```

Comment:
You may also add another case to assert False in this function

Reply:
Fixed.